spec/visual/samples/javascript

```diff
@@ -207,6 +207,13 @@ function* range(from, to)
 `this\n \` \${2} $` is\n ${"sparta".toUpperCase() + function() { return 4; } + "h"}`;
 `hello ${"${re}" + `curs${1}on`}`;
 
+`${
+  'multiline template'
+}`;
```

Comment:
My previous fix had worked with the "1 + 2" example but broke when I tried using a string 😄